R/fullREMIND.R

```diff
@@ -170,12 +171,16 @@ fullREMIND <- function(rev=0) {
     regionmapping2use <- infoConfig$regionmapping
     setConfig(regionmapping = "2b1450bc.csv")
     ## run EDGE-T
-    lapply(c("ConvCase", "ElecEra", "HydrHype", "ConvCaseWise", "ElecEraWise", "HydrHypeWise"),
-           function(x){
+    lapply(strsplit(cartesian(x = c("ConvCase", "ElecEra", "HydrHype", 
+                                    "ConvCaseWise", "ElecEraWise", 
+                                    "HydrHypeWise"),
+                              y = paste0('SSP', c(1, 2, 5))),
```

Comment:
maybe include SDP and SSP2-ARIADNE while we are at it?

Reply:
Is `SSP2-ARIADNE` available yet?

Reply:
ok,
```suggestion
                              y = c('SDP', paste0('SSP', c(1, 2, 5)))),
```
and we leave aside SSP2-ARIADNE